packages/cli/package.json

```diff
@@ -13,6 +13,7 @@
     "lint": "gulp lint",
     "build": "gulp build",
     "test": "gulp test lint",
+    "prepublishOnly": "ls npm-shrinkwrap.json",
```

Comment:
?

Reply:
I assumed this is here to exit non-0 when shrinkwrap file is missing.

Reply:
Correct. If you try to publish CLI but forgot to make a shrinkwrap, then it will fail with `ls: npm-shrinkwrap.json: No such file or directory`